### DISCUSSION TITLE
Default Text Trait

### DISCUSSION THREAD

**Discussion by RakulAgn** [GrapesJS/grapesjs#5576] [General]
i have some doubt on default text traits. in text trait by default the values are updating in onBlur Method Instead in need to replace with onChange Method so it will trigger the changes on every keystroke 

this is what i tried but this is not working as aspected is there any possible way to change by default 

i know some alternative methods to by creating trait on Trait Manager but i am preferring to do with the default option by extendsing the method and changing the event type
![DefaultTextTrait](https://github.com/GrapesJS/grapesjs/assets/119593989/ed2926bd-461d-48eb-84d0-8d21235119eb)

**Comment by artf** [comment]
@RakulAgn 🤦‍♂️... you already opened the same discussion here https://github.com/GrapesJS/grapesjs/discussions/5572
